releases/latest/snaps.yaml

```diff
@@ -5,12 +5,12 @@ packages:
 - name: charmed-mysql
   push_channel: 8.0/edge
   revision: 109
+- name: grafana-agent
+  push_channel: latest/stable
+  revision: 82
```

Comment:
@lucasgameiroborges Is it the one time reordering incident? 

P.S. the snaps order doesn't looks alphabetically sorted nowadays.
Maybe worth to add sorting on dumping into the file.